lib/web/desktop.go

```diff
@@ -103,17 +102,35 @@ func createDesktopConnection(
 	//
```

Comment:
In the comment above
> Pick a random Windows desktop service as our gateway

is this still random? should it be?

Reply:
Ah it should be, changed to shuffle it, I believe the database server selection is also shuffled.